test/test_utils/__init__.py

```diff
@@ -1210,6 +1210,17 @@ def get_cuda_version_from_tag(image_uri):
     if all(keyword in image_uri for keyword in cuda_str):
         cuda_framework_version = re.search(r"(cu\d+)-", image_uri).groups()[0]
 
+    if not cuda_framework_version and "gpu" in image_uri:
+        response_output = json.loads(run(f"docker inspect {image_uri}", hide=True).stdout)
+        env_vars = response_output[0]['ContainerConfig']['Env']
+        cuda_version_env_variables = [env_var for env_var in env_vars if env_var.startswith("CUDA_VERSION=")]
+        if not cuda_version_env_variables:
+            raise RuntimeError(f"Image {image_uri} does not have an Env variable called CUDA_VERSION")
```

Comment:
Why is this check being conducted in a `get` function for the CUDA version being obtained from an image tag?

This function is not a test, and the lack of `CUDA_VERSION` as an env variable in the image should not raise a `RuntimeError` here.